src/Microsoft.AspNetCore.All/build/PublishWithAspNetCoreTargetManifest.targets

```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(PublishWithAspNetCoreTargetManifest)'=='true'">
-    <TargetManifestFiles>$(TargetManifestFiles);$(MSBuildThisFileDirectory)manifest.win7-x64.xml;$(MSBuildThisFileDirectory)manifest.win7-x86.xml;$(MSBuildThisFileDirectory)manifest.osx-x64.xml;$(MSBuildThisFileDirectory)manifest.linux-x64.xml</TargetManifestFiles>
+    <TargetManifestFiles>$(TargetManifestFiles);$(MSBuildThisFileDirectory)aspnetcore-store-__MANIFEST_VERSION__-win7-x64.xml;$(MSBuildThisFileDirectory)aspnetcore-store-__MANIFEST_VERSION__-win7-x86.xml;$aspnetcore-store-__MANIFEST_VERSION__-osx-x64.xml;$(MSBuildThisFileDirectory)aspnetcore-store-__MANIFEST_VERSION__-linux-x64.xml</TargetManifestFiles>
```

Comment:
I am seeing the errors during publish, seems like globs do not work:

```
 (RunResolvePublishAssemblies target) -> 
         C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018: The "ResolvePublishAssemblies" task failed unexpectedly. [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018: System.Xml.XmlException: Cannot open 'file:///C:/Users/johluo/.nuget/packages/microsoft.aspnetcore.all/2.0.0-preview2-t0048339c4/build/*.xml'. The Uri parameter must be a file system relative or absolute path. ---> System.ArgumentException: Illegal characters in path. [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018: Parameter name: path [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.IO.Path.GetFullPath(String path) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.Xml.XmlSystemPathResolver.GetEntity(Uri uri, String role, Type typeOfObjectToReturn) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    --- End of inner exception stack trace --- [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.Xml.XmlSystemPathResolver.GetEntity(Uri uri, String role, Type typeOfObjectToReturn) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.Xml.XmlTextReaderImpl.FinishInitUriString() [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.Xml.XmlTextReaderImpl..ctor(String uriStr, XmlReaderSettings settings, XmlParserContext context, XmlResolver uriResolver) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.Xml.XmlReader.Create(String inputUri, XmlReaderSettings settings) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at System.Xml.Linq.XDocument.Load(String uri, LoadOptions options) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at Microsoft.NET.Build.Tasks.StoreArtifactParser.Parse(String filterFile) [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at Microsoft.NET.Build.Tasks.ResolvePublishAssemblies.ExecuteCore() [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at Microsoft.NET.Build.Tasks.TaskBase.Execute() [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [C:\gh\tp\TestStore\TestStore.csproj]
       C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006127\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.Publish.targets(252,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__25.MoveNext() [C:\gh\tp\TestStore\TestStore.csproj]
```

Took a brief look but couldn't figure out why MSBuild isn't working as I expected. Will revert to using explicit file names.